.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -1,10 +1,9 @@
-name: 🪲 Content bug or inaccuracy
+name: 🪲 Content bug or inaccuracy with Platform docs (_not_ API reference docs)
```

I don't think "Platform" is very clear.

```suggestion
name: 🪲 Content bug or inaccuracy with non-API docs
```